packages/cxl-lumo-styles/scss/icons.scss

```diff
@@ -0,0 +1,26 @@
+// Unicode character map for the icons
+$icons: (
+  quote-right: '\\275e'
```

Comment:
OK so ideally we'd want to use the same asset everywhere, not general unicode here and SVG there.

@freudFlintstone do you think you could take a look at https://github.com/vaadin/vaadin-lumo-styles/blob/v1.6.0/font-icons.html and propose an adoption strategy here.

Something like

```css
    @font-face {
      font-family: 'cxl-lumo-icons';
      src: url(data:application/font-woff;charset=utf-8;base64,<encoded>) format('woff');
      font-weight: normal;
      font-style: normal;
    }
```

where we gotta figure out how to ideally use Vaadin's build chain to use our source SVGs to get an base64-encoded WOFF string.

Feasible?

Maybe a crude hack could also be

* fork vaadin/vaadin-lumo-styles
* branch it
* add our icons
* build a custom WOFF asset in forked branch
* copy asset here
* write docs on the "how" for other people

It feels like bringing a whole Gulp-based toolchain in here might be too much work for too little benefit :thinking:

Reply:
Vaadin build chain has problems https://github.com/vaadin/vaadin-lumo-styles/issues/85

Reply:
At minimum, this variable needs to become a document level CSS variable to follow upstream system.

Reply:
Ok, following that minimum route, this is the situation:

I changed the name in the map to follow the vaadin icons font, using the --lumo-icons custom property prefix. 

That means two steps to add custom icons:
- Add svg to `packages/cxl-lumo-styles/src/icons.js` with id="--lumo-icons-<icon>"
- Add unicode mapping to 'packages/cxl-lumo-styles/scss/icons.scss` with new "--lumo-icons-<icon>" custom property;

While it's annoying to manage assets separately, usage is seamlessly integrated.
Usage would also be simple and identical with a new gulp process, and assets would be unified, but adding icons would still be a manual process and the entire thing would be more obscure and have added complexity to the project. Let me know if you think we really should pursue that path.

Reply:
I have Vaadin `gulp icons` working on Gulp 4, we have broken through here and will move forward tomorrow :tada: